extras/free-books.md

```diff
@@ -87,6 +87,9 @@ Name | Author(s)
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author(s) 
+:-- | :--: 
+[Linear Algebra](https://www.amazon.com/Linear-Algebra-2nd-Kenneth-Hoffman/dp/0135367972) | Kenneth M. Hoffman
```

Comment:
Do you have a link to the free version? Otherwise this should go in the paid-books file. Thanks!